examples/cython/sycl_buffer/src/use_sycl_buffer.hpp

```diff
@@ -29,8 +29,9 @@
 
 #pragma once
 
-#include <CL/sycl.hpp>
 #include <algorithm>
+#include <cstdint>
```

Comment:
This is unused in the file

std::size_t is (surprisingly) [defined in `cstddef`](https://en.cppreference.com/w/cpp/types/size_t)